_utility/send_gh_comment_validation.sh

```diff
@@ -0,0 +1,105 @@
+# Define help function
+function help(){
```

Comment:
This file has been autogenerated, I rather prefer leave it like it is, so if someone needs to add a new parameter in the future its as easy as autogenerate it in the webpage that I wrote in the comment in the header description